AK/StdLibExtras.h

```diff
@@ -68,9 +68,16 @@ constexpr T&& move(T& arg)
     return static_cast<T&&>(arg);
 }
 
+template<typename T, typename... Args>
+constexpr T* construct_at(T* p, Args&&... args)
+{
+    return new (const_cast<void*>(static_cast<void const*>(p))) T(forward<Args>(args)...);
```

Comment:
But if `AK_REPLACED_STD_NAMESPACE` is `AK::replaced_std`, this won't be in std namespace.

Reply:
makes sense,
I think I can get around that by always `using` the `std::` version in AK?
(I hate this function, and cant wait for the c++26 being officially implemented everywhere)

Reply:
Also CC @nrdmn, as this affects your PR/commit